### PR TITLE
fix(example): fix koa example

### PR DESCRIPTION
**Pull request by umuoy1**
## Which problem is this PR solving?

- koa example wasn't to be able to run correctly.

## Short description of the changes

- fixes the status code for spans
- sets the serviceName for the example

### CODE REVIEW

File: examples/koa/client.js [hunk 1]
```diff
@@ -13,10 +13,10 @@ function makeRequest() {
   api.context.with(api.trace.setSpan(api.ROOT_CONTEXT, span), async () => {
     try {
       const res = await axios.get('http://localhost:8081/run_test');
-      span.setStatus({ code: api.StatusCode.OK });
+      span.setStatus({ code: api.SpanStatusCode.OK });
```
Comment (blumamir) [inline examples/koa/client.js:16]: mmm, I think that we should actually delete this line.
span status code should be `UNSET` which is the default.

[from the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#set-status):
> Generally, Instrumentation Libraries SHOULD NOT set the status code to Ok, unless explicitly configured to do so. Instrumentation Libraries SHOULD leave the status code as Unset unless there is an error, as described above.

File: examples/koa/tracer.js [hunk 2]
```diff
@@ -9,17 +9,23 @@ const { NodeTracerProvider } = require('@opentelemetry/sdk-trace-node');
 const { SimpleSpanProcessor } = require('@opentelemetry/sdk-trace-base');
 const { JaegerExporter } = require('@opentelemetry/exporter-jaeger');
 const { ZipkinExporter } = require('@opentelemetry/exporter-zipkin');
+const { Resource } = require("@opentelemetry/resources")
+const { SemanticResourceAttributes } = require("@opentelemetry/semantic-conventions")
```
Comment (blumamir) [inline examples/koa/tracer.js:13]: The `require`s above use a single quote, can be nice to follow the existing style: `require('@opentelemetry/resources')`

**Review by blumamir — APPROVED** [review]

File: examples/koa/client.js (hunk 1 above)
Comment (legendecas) [reply examples/koa/client.js:16]: Isn't this example an end-user example?

**Review by legendecas — APPROVED** [review]
Thank you for your contribution! LGTM % the style nits mentioned by @blumamir.

File: examples/koa/tracer.js (hunk 2 above)
Comment (umuoy1) [reply examples/koa/tracer.js]: done

File: examples/koa/client.js (hunk 1 above)
Comment (legendecas) [reply examples/koa/client.js:16]: @blumamir would you mind shedding your light on this again? I believe this is an end-user example so setting the status code here should be fine.

Comment (dyladan) [reply examples/koa/client.js:16]: @legendecas is correct the end user should set the span status and this example is fine IMO

**Review by dyladan — APPROVED** [review]

**Pull request merged**